README.md

```diff
@@ -1 +1,59 @@
-# ODP
\ No newline at end of file
+# E2E-Test Framework
+
+##1. How to run the test against a) Local sandbox and b) Remote CDAP instance.
+
+### Local sandbox:
+
+Provide the local sandbox URL in [ConnectionParameters.properties](src/main/resources/ConnectionParameters.properties) file.
+
+In the ConnectionParameters.properties file, provide the local sandbox URL in
+[ConnectionParameters.properties](src/main/resources/ConnectionParameters.properties)
+
```

Comment:
Line no 7 and Line no 9, 10 look pretty similar to me. Can we consolidate them?

Reply:
Done.